### PR TITLE
S3 server side encryption

### PR DESCRIPTION
**Pull request by jacekszubert**
S3 server side encryption adde as an option (AES256 support so far).

### CODE REVIEW

**Review by xaf — CHANGES_REQUESTED** [review]
Same as for pull request #3 - please change:
- Rebase cleanly, your pull requests should not contain a "Merge pull request #1 from XaF/master" commit, but be based on the master of this repository
- Add your signoff to your commits (git commit -s)

Once done, I'll merge this pull request.
Thanks a lot for your contribution :o)

**Comment by jacekszubert** [comment]
:+1:

File: dynamodb-bnr.py [hunk 1]
```diff
@@ -204,6 +204,8 @@ def run(self):
               help='Define the AWS DynamoDB region')
 @click.option('--s3/--no-s3', default=False,
               help='Activate S3 mode')
+@click.option('--s3-sse', default=False,
```
Comment (xaf) [inline dynamodb-bnr.py:207]: @click.option('--s3-sse', is_flag=True, default=False,

**Review by xaf — APPROVED** [review]
Sorry, did not catch that before.
I'l apply the changes and merge.

**Comment by xaf** [comment]
Merged in commit e63646e3b8691fa006f2a0c265cecf6c1e502e6c
Thanks

**Pull request closed without merging**